code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -176,6 +176,7 @@ Difficulty: Medium
 	hitsound = 'sound/weapons/sear.ogg'
 	var/storm_type = /datum/weather/ash_storm
 	var/storm_cooldown = 0
+	var/static/list/excluded_areas = list(/area/reebe/city_of_cogs)
```

Comment:
lame

Reply:
This would literally kill everyone on the z_level since its all 1 area

Reply:
Yes.